README.md

```diff
@@ -13,18 +13,27 @@ JWST Calibration Pipeline
 Installation
 ------------
 
-The `jwst` package can be installed into a virtualenv or conda environment via `pip`.  We recommend creating a fresh environment with only python installed.  If using conda, make sure you have a recent version of Anaconda or Miniconda installed.  Via conda:
+The `jwst` package can be installed into a virtualenv or conda environment via `pip`.  In general, we recommend creating a fresh environment with only python installed.  If using conda, make sure you have a recent version of Anaconda or Miniconda installed.  From a bash shell enter:
+## Step 1. Create a Conda Environment ##
 
-    conda create -n jwst_env python
-    conda activate jwst_env
+    conda create -n <env_name> python
+    conda activate <env_name>
 
-### Installing for end-users ###
+## Below are directions on how to install three different versions of the pipeline.
```

Comment:
Change to "directions for 3 methods to install the pipeline".  All install the same version (same code) of the pipeline.